lib/service.js

```diff
@@ -81,6 +93,26 @@ class Service {
     });
   }
 
+  /**
+   *
+   * @param {string|string[]} files
+   * @returns {Promise}
+   */
+  uploadFiles(files) {
+    const formData = { attachments: [] };
+
+    for(const file of files) {
+      const stream = fs.createReadStream(file);
```

Comment:
@tttmaximttt - we would rather accept an array of "Buffers" and "ReadStreams".   We should not be creating the readstream from a file path ourselves.  The Developer (SDK user) should create his/her own buffers/readstreams from files (or anything else for that matter), and then pass that to us.  Add validation to ensure we only are excepting an arrays that have Buffers and ReadStreams.  
<img width="1155" alt="class__aws_s3_ _aws_sdk_for_javascript" src="https://cloud.githubusercontent.com/assets/3407325/25812926/7f4cfd28-33dd-11e7-9c86-4fa494b257b6.png">